README.md

```diff
@@ -36,6 +36,10 @@ It wraps a given `ReadableStreamInterface` and exposes its plain data through
 the same interface.
 
 ```php
+<?php
+
+require __DIR__ . '/vendor/autoload.php';
+
 $stdin = new React\Stream\ReadableResourceStream(STDIN);
 
 $stream = new ControlCodeParser($stdin);
```

Comment:
```suggestion
$stream = new Clue\React\Term\ControlCodeParser($stdin);
```

Reply:
Should be added now, thank you very much @clue 🥳